.github/workflows/ci.yml

```diff
@@ -36,3 +36,35 @@ jobs:
         working-directory: ./frontend
       - run: yarn format
         working-directory: ./frontend
+
+  backend:
+    strategy:
+      matrix:
+        node: ["14.x"]
+        os: [ubuntu-latest]
+    runs-on: ${{ matrix.os }}
+    steps:
+      - uses: actions/checkout@v1
+      - uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node }}
+      - name: Cache
+        id: cache
+        uses: actions/cache@v2
+        with:
+          path: ./backend/cache/cache.json
+          # Update the version if cache format changes
+          key: cache-json-v-1-${{ github.run_id }}
+          restore-keys: |
+            cache-json-v-1-
+      - run: yarn --frozen-lockfile
+        working-directory: ./backend
+      - run: yarn lint
+        working-directory: ./backend
+      - run: yarn typecheck
+        working-directory: ./backend
+      - run: yarn start
```

Comment:
We don't need to refresh data.json on every commit but rather on daily schedule. So I would remove this bit and tweak CI CRON job.

Reply:
I actually think we should keep this step as it is crucial for this code to work. The `data.json` file is overwritten but not committed and the cache would be reused from the master branch on any PR that doesn't yet have its own.

I can add a flag like `--fetch-only` that does not overwrite `data.json`

Reply:
Maybe once we have eslint & tests it is less needed. Let's revisit this topic then :)

Reply:
Removed this step